### PR TITLE
#3943 [Manager] Reverse the order of the buttons in create workspace page

### PR DESCRIPTION
**Pull request by eechih**
# Checklist 
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3943 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Workspace/Create/WorkspaceQuick.js [hunk 1]
```diff
@@ -458,6 +460,12 @@ const WorkspaceQuick = props => {
                           <StyledPaper>
                             {getStepContent(index, values)}
                           </StyledPaper>
+                          <StyleButton
+                            disabled={activeStep === 0}
+                            onClick={() => handleBack(activeStep)}
+                          >
+                            Back
+                          </StyleButton>
```
Comment (eechih) [inline ohara-manager/client/src/components/Workspace/Create/WorkspaceQuick.js]: BACK 及 NEXT 按鈕的順序應該要遵循 MUI 的標準設計

File: ohara-manager/client/src/components/Layout/AppBar.js [hunk 2]
```diff
@@ -103,7 +103,11 @@ const AppBar = () => {
           <Tooltip title="Workspace list" placement="right">
             <IconButton
               className="workspace-list item"
-              onClick={toggleWorkspaceList}
+              onClick={() =>
+                isEmpty(workspaces)
+                  ? setIsNewWorkspaceOpen(true)
+                  : toggleWorkspaceList()
+              }
```
Comment (eechih) [inline ohara-manager/client/src/components/Layout/AppBar.js]: 還沒有建立任何 workspace 前，按下 AppBar 的 list workspace 的按鈕，應該引導使用者去建立新的 workspace

**Comment by saivirtue** [comment]
I think we don't "test" by QA now... 😄

**Comment by eechih** [comment]
> I think we don't have "tests" by QA now... 😄

喔喔

**Pull request merged**